gradle-plugin/src/main/java/com/google/cloud/tools/dependencies/gradle/LinkageCheckTask.java

```diff
@@ -307,13 +307,25 @@ private static DependencyGraph createDependencyGraph(ResolvedConfiguration confi
 
       DependencyPath parentPath = item.getParentPath();
 
-      // parentPath is null for the first item
-      DependencyPath path =
-          parentPath == null
-              ? new DependencyPath(artifactFrom(node))
-              : parentPath.append(dependencyFrom(node));
+      DependencyPath path = null;
```

Comment:
Just reminder to define this in the loop if it is no longer needed outside.

Reply:
This variable is needed outside.